backend/src/routes/v1/approvals.ts

```diff
@@ -132,6 +137,18 @@ export const postApprovalResponse = [
       version[field] = choice as ApprovalStates
       await version.save()
 
+      if (approval.approvalType === ApprovalTypes.Manager && choice === ApprovalStates.Accepted) {
+        if (version.reviewerApproved !== ApprovalStates.Accepted) {
+          throw BadReq(
+            {
+              code: 'bad_request_type',
+              version: versionDoc.managerApproved,
+            },
+            'Managers cannot approve model versions until it has been accepted by a technical reviewer'
```

Comment:
Can we get these role names from the schema itself? "Manager" and "Technical Reviewer" might not be the right definition if the schema changes.

Reply:
Done